### DISCUSSION TITLE
Reverification error - payload too large with 1.6M image - resized to 190K = accepted

### DISCUSSION THREAD

**Discussion by maueroats** [community/community#102077] [GitHub Education]
Keywords: Github Education, reverification silently fails, no progress bar on submission, problem solved 

### Hi everyone,

Attempting to upload a 1.6MB image file results in a "Payload Too Large" message being logged to the console. No progress meter appears. 

![image](https://github.com/community/community/assets/11153074/d334a79f-19c2-496f-9c4d-f7c18e536b02)

This software is garbage... after simply resizing the image to 1024x768 at quality 80, it uploads and I am accepted.

**Comment by a-calf** [comment]
Lower the pixel count to meet the requirement of 1024 * 768
